ui/package.json

```diff
@@ -12,6 +12,8 @@
     "prepare": "cd .. && husky install ui/.husky",
     "build:lib": "pnpm -r --filter @pingcap/tidb-dashboard-lib build",
     "dev": "pnpm build:lib && pnpm -r --parallel dev",
+    "dev:op": "pnpm build:lib && pnpm -r --parallel --filter @pingcap/tidb-dashboard-for-op... dev",
+    "dev:dbaas": "pnpm build:lib && pnpm -r --parallel --filter @pingcap/tidb-dashboard-for-dbaas... dev",
```

Comment:
hi @shhdgit , can you attach some doc links to present the `...` meaning in the PR description. Although we can google it, it is more convenient for reviewers.

Reply:
That's right! Thanks!